content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+{empty} +
+*Software Updates* +
+Manages automatic package updating, and allows for the manual installation of updates
+{empty} +
+*Subscriptions* +
+Manages subscriptions attached to the host
+{empty} +
+*Terminal* +
+Provides an interactive shell
+--
+
+Next, let's proceed to a more detailed examination of each module. Please click through the web console on your host, as you read the explanations, here.
+
+=== Section 2: System Module
+
+image::cockpit_system.png[caption="Figure 2: ", title='Web Console System Module', 500]
+
+The system module is the main screen that you see, when you first login to the web console. It contains an overview of the system, including the below sections:
+
+* hardware type
+** detailed system hardware data is available, from the link on the main page
+* asset tag
+* machine ID
+* operating system
+** when updates to the system are available, an "updates available" link will be present
+* SSH keys
+** key fingerprints for all created 'root' keys
+* host name
+** the system's name may be changed, and an alias may be added
+* domain
+** the system may be added to an LDAP/AD domain
+* time and date
+** time zone and automatic or manual time and date setting can be done
+* performance profile
+** the system's performance profile may be changed
+
+Additionally, there are graphs that show the system's CPU, memory, disk, and network utilization.  Each graph may be enlarged, by clicking its associated link.
+
+=== Section 3: Logs Module
+
+image::cockpit_01_logs.png[caption="Figure 3: ", title='Web Console Logs Module', 500]
+
+The logs module provides date-sorted view of recent system log entries.  You can change the date or apply basic filters using the pull-down menus at the top of the page. 
+
+=== Section 4: Storage Module
+
+image::cockpit_02_storage.png[caption="Figure 4: ", title='Web Console Storage Module', 500]
+
+The storage module provides graphs of current storage performance.  It also lists local and network-mounted filesystems, plus recent storage-related log entries.
+
+=== Section 5: Networking Module
+
+image::cockpit_03_networking.png[caption="Figure 5: ", title='Web Console Networking Module', 500]
+
+The network module contains performance graphs similar to the storage module.  Additionally, it shows the configuration of your network interfaces, identifies unused interfaces, and provides access to, and control of, the system firewall.
+
+=== Section 5: Podman Containers Module
+
+image::cockpit_04_podman.png[caption="Figure 6: ", title='Web Console Podman Containers Module', 500]
+
+This module provides control of local podman containers and their corresponding images.
+
+=== Section 6: Virtual Machines Module
+
+image::cockpit_05_vm.png[caption="Figure 7: ", title='Web Console Virtual Machines Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
```

Comment:
Line 141 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics: